### PR TITLE
walletStatus$ fixed

### PR DESCRIPTION
**Pull request by maciejka**

### CODE REVIEW

File: src/blockchain/wallet.ts [hunk 1]
```diff
@@ -23,43 +23,50 @@ export const accepted$ = interval(500).pipe(
...
-      return from(['missing' as WalletStatus]);
-    }),
   ),
-).pipe(
+  tap(console.log),
   shareReplay(1),
 );
 
-walletStatus$.subscribe();
+// walletStatus$.subscribe(console.log);
```
Comment (peculiarity) [inline src/blockchain/wallet.ts]: just remove this one and the tap

File: src/blockchain/wallet.ts [hunk 2]
```diff
@@ -23,43 +23,50 @@ export const accepted$ = interval(500).pipe(
...
-export const walletStatus$: Observable<WalletStatus> = merge(
-  combineLatest(
-    account$,
-    accepted$
-  ).pipe(
-    map(([account, hasAcceptedToS]) =>
+const connectToWallet$: Subject<number> = new Subject();
+
+export function connectToWallet() {
+  connectToWallet$.next(1);
```
Comment (peculiarity) [inline src/blockchain/wallet.ts]: Why do you provide 1? It really doesn't matter as far as I know.

Comment (peculiarity) [reply src/blockchain/wallet.ts]: Also why do we need to provide type to the Subject?

Comment (maciejka) [reply src/blockchain/wallet.ts]: No need to expose subjects all over the place.

File: src/blockchain/wallet.ts (hunk 1 above)
Comment (maciejka) [reply src/blockchain/wallet.ts]: Ok

**Pull request merged**